src/main/drivers/compass.h

```diff
@@ -20,10 +20,11 @@
 #include "sensor.h"
 
 typedef struct magDev_s {
-    sensorInitFuncPtr init;                                 // initialize function
-    sensorReadFuncPtr read;                                 // read 3 axis data function
+    sensorMagInitFuncPtr init;  // initialize function
+    sensorMagReadFuncPtr read;  // read 3 axis data function
```

Comment:
I'd prefer to use the suffix `FnPtr` rather than `FuncPtr`. Eventually I'd like to change them all.